recipes/hasasia/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "hasasia" %}
+{% set version = "1.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 87ab202bb6c07b24922169fc12115f9fe074c9d6ae33df08b91cb3c4ca48436f
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - numpy >=1.14
+    - scipy >=1.0
+    - astropy >=3.0
+
+test:
+  imports:
+    - hasasia
```

I just noticed that `hasasia.__init__` basically doesn't do anything. Can you add some more imports here that verify the package isn't broken. Maybe like?
```suggestion
    - hasasia
    - hasasia.sensitivity
```

---

Okay, I've now imported all of the submodules.